lib/views/help/contact.html.erb

```diff
@@ -132,16 +132,31 @@
         </ul>
     </div>
 
+    <h2 class="contact-page__goal" id="wdtk">
```

Comment:
```suggestion
    <h2 class="contact-page__goal" id="report-data-breach">
```

Reply:
Oops, should be `goal5`; looks like I missed this one. Good spot!

Reply:
Oh, no, you are right! 🤦‍♂️